py/fiberassign/fba_launch_io.py

```diff
@@ -2137,7 +2138,8 @@ def qa_print_infos(
             ["GAIA_PHOT_G_MEAN_MAG)", "min(LS-R-FIBTOTMAG)"],
         ):
             magmin, color = "-", "k"
-            keep = (np.isfinite(mag)) & (mag > 0)
+            # np.nan_to_num: NaN -> 0, so keep=False.
+            keep = (np.nan_to_num(mag) > 0)
```

Comment:
If mag==np.inf, then the previous test would be False, while this new test would be True.  np.isfinite already handles NaN (returns False without complaining), so is this change needed?

Reply:
Good catch.  np.nan_to_num does accept a value to set pos/neg inf to -- I suppose setting those both to zero would achieve what we want.

This PR was cosmetic, to suppress a warning coming from numpy, because when any of the mags are NaN, the comparison (mag>0) warns, even though we're then going to & it with False.

Reply:
(I just pushed 0de4b93 to do this)